resources/views/themes/default1/front/auth/login-register.blade.php

```diff
@@ -203,14 +203,29 @@
 
                                                         </div>
                                                     </div>
-                                                    <div class="form-group col-lg-6">
-                                                        <input type="submit" value="Login" id="submitbtn" class="btn btn-primary pull-right mb-xl" data-loading-text="Loading...">
-                                                        <!-- <button type="button" class="btn btn-primary mb-xl next-step float-right" name="sendOtp" id="login" onclick="loginUser()">
-                                                                    Send Email
-                                                        </button> -->
+                                                    <div class="form-group col-12 ">
+                                                    <div class="row">
+                                                        <div class="form-group col">
```

Comment:
Shorter this code. Write a single button code instead of this huge

Reply:
Actually there are 4 logins if i somehow replicate the same link for 4 buttons , still if there is the need of 2 ,3, or 4 buttons then it is not possible for me to display all of them with only one button ,